lighthouse-extension/gulpfile.js

```diff
@@ -155,6 +162,33 @@ gulp.task('browserify', cb => {
   runSequence('browserify-lighthouse', 'browserify-other', cb);
 });
 
+gulp.task('compilejs', () => {
+  const opts = {
+    mangle: false, // Preserve  array notation property access. Needed for artifact name lookups.
```

Comment:
It's pretty tricky to track down where things break down, but without `mangle: false`, the audit names get rewritten:

<img width="486" alt="screen shot 2017-02-21 at 8 55 57 am" src="https://cloud.githubusercontent.com/assets/238208/23179933/c8068f08-f813-11e6-9bd2-59e5f568be32.png">
<img width="833" alt="screen shot 2017-02-21 at 8 56 25 am" src="https://cloud.githubusercontent.com/assets/238208/23179934/c807c5b2-f813-11e6-901b-aae1e1dd2f83.png">

Hard to say if it also creates other errors.